src/components/Header/TopNav/LogoLink.js

```diff
@@ -22,40 +18,45 @@ const StyledLink = styled(Link)`
     width: ${remcalc(54)};
   }
 `
+const HomepageLogo = styled.img`
+  width: 49px;
+  height: 36px;
+  margin-top: ${remcalc(6)};
+`
+
+const LogoLink = ({ slug, isServicePage, isSpecialityPage, isHomePage }) => {
+  const renderSvg = isServicePage || isSpecialityPage || isHomePage
+
+  const fillColorInitial = logoColors[slug] || logoColors['default']
 
-const LogoLink = ({ path = '/' }) => {
-  const isServicePage = getIsServicePage(path)
-  const isSpecialityPage = path.includes('speciality')
+  const fillColorHover = isSpecialityPage
+    ? logoColors.specialityHover
+    : logoColors.defaultHover
 
-  const serviceTitle = isServicePage ? path.match(servicesRegExp)[0] : null
-  const service = isSpecialityPage ? getService(path) : serviceTitle
-  const speciality = isSpecialityPage ? getSpeciality(path) : null
+  const textColor = isSpecialityPage
+    ? logoColors['specialityText']
+    : logoColors['defaultText']
 
-  let originalFillColor = isSpecialityPage
-    ? logoColors[service][speciality]
-    : logoColors['default']
-  const [fillColor, setFillColor] = useState(originalFillColor)
+  const [fillColor, setFillColor] = useState(fillColorInitial)
 
   return (
     <Fragment>
-      {isSpecialityPage || isServicePage ? (
-        <StyledLink
-          to="/"
-          title="Return to Homepage"
-          onMouseEnter={() =>
-            setFillColor(
-              logoColors[isServicePage ? 'defaultHover' : 'specialityHover']
-            )
-          }
-          onMouseLeave={() => setFillColor(originalFillColor)}
-        >
-          <ServiceSpecialityLogo
-            fillColor={fillColor}
-            textColor={
-              logoColors[isServicePage ? 'defaultText' : 'specialityText']
-            }
-          />
-        </StyledLink>
+      {renderSvg ? (
```

Comment:
I think it's more readable to have an `if` instead of nested ternary operators

```
if(renderSvg) {
    return (
        isServicePage || isSpecialityPage ? (
          <StyledLink
            to="/"
            title="Return to Homepage"
            onMouseEnter={() => setFillColor(fillColorHover)}
            onMouseLeave={() => setFillColor(fillColorInitial)}
          >
            <ServiceSpecialityLogo
              fillColor={fillColor}
              textColor={textColor}
            />
          </StyledLink>
        ) :
          <HomepageLogo src={HomepageSvg} alt="yld logo" />
    )
}

return (
    <Link to="/" title="Return to Homepage">
        <img role="link" height="48" src={logo} alt="yld logo" />
    </Link>
)
```